Maths/CollatzSequence.js

```diff
@@ -0,0 +1,11 @@
+export function collatz (n) {
+  while (n > 1) {
+    if (n % 2 === 0) {
+      n = n / 2
+    } else {
+      n = 3 * n + 1
+    }
+  }
+
+  return n
```

Comment:
This program will either never halt or return 1. You should do something more useful with the return value, like counting the number of steps before it reaches the 1, 2, 4 loop.